feeds.xsh

```diff
@@ -0,0 +1,24 @@
+import asyncio
+import threading
+import time
+from feedy import Feedy
+
+app = Feedy('feeds.shelve')
+
+@app.add('https://stackoverflow.com/feeds/tag?tagnames=xonsh&sort=newest')
+def stackoverflow_question(info, body):
```

Comment:
It looks (from [this post](https://gitter.im/xonsh/xonsh?at=58e008e708c00c092a3b6161)) like when the bot is first started up, it will notice all the old xonsh posts, which, in the event that we have to restart the bot (has happened before...), would cause all the old questions to pop up again.  Is there an easy way to tell the bot only to post about "new" things?  Either by keeping a timestamp of when the bot was started and comparing against `published_at`, or by some mechanism within `feedy`?